assets/donators.json

```diff
@@ -375,8 +375,12 @@
     "uuid": "0fcfda42-fbdc-40c1-af36-926aa1ed025e",
     "type": "-1"
   },
+  {
+    "uuid": "f14c348d-4e38-415a-aba5-8688c71d7288",
+    "type": "-1"
+  },
   {
     "uuid": "75e17a15-c1bc-4167-bda4-e7a63be0f465",
     "type": "-1"
-  }
```

Comment:
wth is this

Reply:
idk

Reply:
![image](https://user-images.githubusercontent.com/38266782/88405533-6e299580-cd9d-11ea-9e9b-105c99261e39.png)